gef-ui/server/models/application.js

```diff
@@ -21,9 +23,10 @@ export default class Application {
       const exporterPro = getExporter(application.exporterId);
       const coverTermsPro = getCoverTerms(application.coverTermsId);
       const facilitiesPro = getFacilities(id);
+      const eligibilityCriteriaPro = getEligibilityCriteria();
 
-      const all = await Promise.all([exporterPro, coverTermsPro, facilitiesPro]);
-      [application.exporter, application.coverTerms, application.facilities] = [...all];
+      const all = await Promise.all([exporterPro, coverTermsPro, facilitiesPro, eligibilityCriteriaPro]);
+      [application.exporter, application.coverTerms, application.facilities, application.ecs] = [...all];
```

Comment:
Not for this PR, just a note - in TFM we will eventually be calling a single endpoint in Central API to "get a gef deal" that will do all of this so we get everything in one response. GEF could then consume this endpoint instead of doing these promises